### PR TITLE
Breaking changes in 1.3

### PR DESCRIPTION
**Pull request by teobugslayer**

### CODE REVIEW

File: breaking-changes.md [hunk 1]
```diff
@@ -11,7 +11,20 @@ This document describes the critical breaking changes and suggested workarounds,
...
+
+You cannot create NativeScript plugins with Android native code using Eclipse projects. You need to import your Eclipse project into Android Studio, convert it to Gradle build and consume the produced AAR file
+
+You cannot use Apache Ant to build your projects for Android. Starting with NativeScript 1.3, Android builds require Gradle. Run `tns doctor` on the command line to learn more.
+
+`library add` command is deprecated and will be removed completely in one of our next releases (currently scheduled for 1.5) 
+
+NSDecimalNumber is marshalled as Objective-C object wrapper instead of JavaScript number 
+
+Builing {N} projects for Android requires Android SDK 22, Build Tools 22, Android support repository and ANDROID_HOME environment variable being set
```
Comment (rosen-vladimirov) [inline breaking-changes.md]: Android SDK Build-tools, Android Support Repository

**Comment by vchimev** [comment]
Let's order them by severity:
1. Building NativeScript projects for Android ...
2. ... Android builds require Gradle
3. ... NativeScript plugins
...

File: breaking-changes.md (hunk 1 above)
Comment (vchimev) [inline breaking-changes.md]: Buil**d**ing

**Comment by teobugslayer** [comment]
Ping @rosen-vladimirov , @vchimev - fixed spelling and reordered items.

File: breaking-changes.md [hunk 2]
```diff
@@ -11,7 +11,19 @@ This document describes the critical breaking changes and suggested workarounds,
...
+
+`library add` command is deprecated and will be removed completely in one of our next releases (currently scheduled for 1.5).
+
+You cannot create NativeScript plugins with Android native code using Eclipse projects. You need to import your Eclipse project into Android Studio, convert it to Gradle build and consume the produced AAR file.
+
+You cannot use Apache Ant to build your projects for Android. Starting with NativeScript 1.3, Android builds require Gradle. Run `tns doctor` on the command line to learn more.
+
+Builing NativeScript projects for Android requires Android SDK 22, Android SDK Build-tools 22, Android Support Repository and ANDROID_HOME environment variable set.
```
Comment (atanasovg) [inline breaking-changes.md]: Builing -> Building

**Comment by atanasovg** [comment]
IMO we should add a note for tns_modules being a separate npm package as well.

**Comment by Plamen5kov** [comment]
:+1:

**Pull request merged**